dipy/viz/fvtk.py

```diff
@@ -79,6 +79,9 @@
     msg = "Python VTK is not installed"
     warn(msg)
 
+    ren, have_ren, _ = optional_package('dipy.viz.window.ren',
```

Comment:
I mean: if we don't have VTK installed, why don't we just raise an exception in the previous line (line 80)? Why do we bother going ahead and trying to do anything?